src/rules/handlers.ts

```diff
@@ -5,6 +5,7 @@ import https = require('https');
 import express = require("express");
 import { OngoingRequest } from "../types";
 import { RequestHandler } from "./mock-rule-types";
+import { OutgoingHttpHeaders } from "http";
```

Comment:
`http` is actually already imported above - I think if you just replace `OutgoingHttpHeaders` below with `http.OutgoingHttpHeaders` you can drop this entirely. Or you could change the above `http` import to use destructuring too, up to you.